spark/src/test/scala/geotrellis/spark/summary/polygonal/FeatureRDDSpec.scala

```diff
@@ -40,14 +40,14 @@ class FeatureRDDSpec extends FunSpec
       val polygon = Polygon( (2, 2), (4, 6), (6, 2), (2, 2) )
 
       val featureRdd = sc.parallelize(Array(lowerExtent, middleExtent, upperExtent).map { e => Feature(e.toPolygon, e.area) })
-      val result = featureRdd.polygonalSummary(polygon, 0.0)(
-        PolygonalSummaryHandler({ feature: Feature[Polygon, Double] => feature.data })
-                           ({ (polygon, feature) => polygon.intersection(feature.geom).asMultiPolygon.map(_.area).getOrElse(0.0) })
-                           ({ (v1, v2) => v1 + v2 })
-      )
-      val expected = polygon.area - 0.5
-
-      result should be (expected)
+//      val result = featureRdd.polygonalSummary(polygon, 0.0)(
```

Comment:
Same comment as above:

> Is this test worth keeping? It implements a custom PolygonalSummaryHandler that summarizes across vector geometries instead of raster ones. I think in the new API, this would be handled by a separate RDD method and isn't something worth supporting at this time unless we know of a valid use case that's actively used.